ceph/rados/rados_service.py

```diff
@@ -0,0 +1,105 @@
+"""
+   This module contains the methods to perform general ceph cluster modification operations.
+    1. Remove OSD
+    2. Add OSD
+    3. Set osd out
+    3. Zap device path
+"""
+import logging
+
+from ceph.rados.core_workflows import RadosOrchestrator
+
+log = logging.getLogger(__name__)
+
+
+class RadosService(RadosOrchestrator):
+    """
+    Perform general cluster modification operations
+    """
+
+    def run_command(self, cmd: str):
+        """
+        Runs ceph commands  for the action specified
+        Args:
+            cmd: Command that needs to be run
+        Returns:
+            out (Str), err (Str) stdout and stderr response
+        """
+        try:
+            out, err = self.node.shell([cmd])
+        except Exception as er:
+            log.error(f"Exception hit while command execution. {er}")
+            return None
+        return out, err
+
+    def set_osd_devices_unamanged(self, unmanaged: bool) -> bool:
+        """
+        Sets osd device unmanaged as true/false
+        Returns:
+            Pass->true, Fail->false
+        """
+        out, _ = self.run_command(
+            f"ceph orch apply osd --all-available-devices --unmanaged={unmanaged}"
+        )
+        if "Scheduled osd.all-available-devices update" in out:
+            return True
+        return False
+
+    def set_osd_out(self, osd_id) -> bool:
+        """
+        Set osd out
+        Args:
+            osd_id: OSD id
+        Returns:
+            Pass->true, Fail->false
+        """
+        cmd = f"ceph osd out {osd_id}"
+        out, err = self.run_command(cmd)
+        if f"marked out osd.{osd_id}" in err:
+            return True
+        return False
+
+    def remove_osd(self, osd_id) -> bool:
```

Comment:
Can this method be added to `cephadm` instead of here? IMO, we can keep the orch commands together